bigbluebutton-html5/imports/ui/components/video-provider/component.tsx

```
@@ -1135,7 +1136,7 @@ class VideoProvider extends Component<VideoProviderProps, VideoProviderState> {
             role,
           },
         }, `Camera ICE connection state changed: ${connectionState}. Role: ${role}.`);
-
+        throw new Error('WebRTC peer failed');
         this.onWebRTCError(error, stream, isLocal);
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)